Source/Core/DolphinQt/Debugger/CodeDiffDialog.h

```diff
@@ -0,0 +1,65 @@
+// Copyright 2021 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include <QDialog>
+
+#include "Common/CommonTypes.h"
+#include "Core/PowerPC/Profiler.h"
+
+class CodeWidget;
+class QCheckBox;
+class QLabel;
+class QListWidget;
+
+struct Diff
+{
+  u32 addr;
+  std::string symbol;
+  u32 hits;
+
+  bool operator<(const std::string& val) { return symbol < val; }

```

Comment:
This method should be `const` according to the FreeBSD buildbot errors.

Reply:
Changed and pushed. Is there a way I can run the buildbot?
Either way I expect the rest of the changes to happen within this week. I'll post when its all done.

Reply:
I've manually triggered a rebuild.

You seem to have done it wrong, though: `const bool operator<(const std::string& val) { return symbol < val; }` should be `bool operator<(const std::string& val) const { return symbol < val; }` as the function is supposed to be `const`, not the return type of the function (there's also a warning that `'const' type qualifier on return type has no effect`).

Reply:
whoops, pushed.